readme.md

```diff
@@ -77,6 +81,21 @@ pipeline:
   epochs: 200
 ```
 
+An example config for using the Thai tokenizer would look like:
+
+```yaml
+language: th
+pipeline:
+  - name: rasa_nlu_examples.tokenizers.thai_tokenizer.ThaiTokenizer
```

Comment:
if you add `ThaiTokenzier` to `tokenizers/__init__.py` file using the `__all__` trick the import path becomes shorter.

Reply:
Sweet - I never used this before!

Reply:
unit tests are a bit tricky as I have no idea how to validate the input or the output...I am using an example sentence from the `PyThaiNLP` Tutorial and hope for the best 🙈

Reply:
That's fine. If we ever meet a community member from Thailand then we can ask for help. The main thing I care for is that there's at least one sanity check in there that will raise a red flag once we break an obvious example.